src/content/guides/how-to-revoke-token-access/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How to revoke smart contract access to your crypto funds
+description: A how to guide on revoking exploitative smart contract token access
+lang: en
+---
+
+# How to: Revoke smart contract access to your crypto funds
+
+With this guide you will learn how to view a list of all smart contracts you have allowed access to your funds and also how to cancel them. 
+
+Sometimes malicious developers build backdoors into smart contracts that will allow access to the funds of unaware users who interact with their projects. What often happens is that such platforms ask the user for permission to spend an **unlimited number of tokens**.
```

Comment:
Unlimited token access isn't necessarily malicious, but it is lazy and stupid. Careful not to make the point that unlimited tokens === hacker.

Reply:
yeah good point maybe we should expand here why this is done (lower the number of transactions/gas on the user's side) but yeah the practice is kinda weird and i dont want to overcomplicate the article